platform-hub-api/app/controllers/kubernetes/clusters_controller.rb

```diff
@@ -1,52 +1,42 @@
 class Kubernetes::ClustersController < ApiJsonController
 
-  before_action :load_kubernetes_clusters_hash_record
-
   # GET /kubernetes/clusters
   def index
     authorize! :read, :kubernetes_clusters
-    clusters = @kubernetes_clusters.data.map do |c|
-      c.with_indifferent_access.slice(:id, :description)
-    end.sort_by! do |c|
-      c[:id]
-    end
-    render json: clusters
+    clusters = Kubernetes::ClusterService.all_clusters.sort_by(&:name)
```

Comment:
Maybe just `all` instead of `all_clusters` (given it's already the `ClusterService`)?